packages/card/style.module.css

```diff
@@ -1,23 +1,21 @@
 .card {
   --border-color: var(--wpl-neutral-200);
   --primary-text-color: var(--wpl-neutral-700);
-  --secondary-text-color: var(--wpl-neutral-500);
+  --secondary-text-color: #75757a;
   --background-color: var(--wpl-neutral-0);
 
-  border: 1px solid var(--border-color);
   display: flex;
   flex-direction: column;
   flex-grow: 1;
   border-radius: 8px;
-  overflow: hidden;
-  position: relative;
-  background-color: var(--background-color);
-  transition: 0.2s ease;
-  transition-property: border-color, background-color;
```

Comment:
Similar to the border-radius comment, I think removing these styles here causes extra spacing on hover + focus. Kind of nitpicky - not sure if we are already okay with this 
<img width="321" alt="Screen Shot 2022-11-11 at 1 10 10 PM" src="https://user-images.githubusercontent.com/20215887/201402438-799b7b9f-1d68-40ad-85c1-3523df234d47.png">

Reply:
Hmm...still getting the same issue when adding these lines back in 🤔 

![6T6Bs7VeBW](https://user-images.githubusercontent.com/3746694/201403580-f5ccf8b0-58cc-48b4-820a-3c8e7fcf7ea5.png)

Reply:
Likely need to add the outline styles to the a tag or the contentContainer since that is what translates on hover.

Reply:
Moved the `&:focus-within` styles to `.contentContainer`, and the outline moves successfully 👍
https://github.com/hashicorp/react-components/pull/776/commits/519a4e94d2b218d390ab72d1e978794e58c5f7f2

Reply:
Oh yeah looks really good now 😄